Tests/TestingApps/PaywallsTester/PaywallsTester/Data/OfferingsPaywallsViewModel.swift

```diff
@@ -49,30 +68,37 @@ final class OfferingsPaywallsViewModel {
             let paywalls = try await appPaywalls
 
             let offeringPaywallData = OfferingPaywallData(offerings: offerings, paywalls: paywalls)
-
-            self.offeringsPaywalls = .success(
-                offeringPaywallData.paywallsByOffering()
-            )
-
+            let listData = PaywallsData(offeringsAndPaywalls: offeringPaywallData.paywallsByOffering(), offeringsWithoutPaywalls: offeringPaywallData.offeringsWithoutPaywalls())
```

Comment:
nitpick: let's keep line lengths at <= 120 chars so we can control the wrapping a little nicer than github does. I'd recommend setting up Xcode's "Page guide at column 120" to make it easier. 
PaywallsTester is a bit of an exception but the rest of our SDK has swiftlint set up to enforce it

Reply:
Done!